bindata/cert-manager-deployment/controller/cert-manager-tokenrequest-rb.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
     app.kubernetes.io/component: controller
     app.kubernetes.io/instance: cert-manager
     app.kubernetes.io/name: cert-manager
-    app.kubernetes.io/version: v1.18.3
-  name: cert-manager-cert-manager-tokenrequest
+    app.kubernetes.io/version: v1.19.2
+  name: cert-manager-tokenrequest
```

Comment:
Not directly related to changes. But how do we manage the renamed resource in upgrade clusters.